runtime/common/src/lib.rs

```diff
@@ -77,3 +77,37 @@ parameter_types! {
 /// https://research.web3.foundation/en/latest/polkadot/overview/2-token-economics.html#-2.-slow-adjusting-mechanism
 pub type SlowAdjustingFeeUpdate<R> =
     TargetedFeeAdjustment<R, TargetBlockFullness, AdjustmentVariable, MinimumMultiplier>;
+
+parameter_types! {
+    /// Time to execute an empty block.
+    /// Calculated by multiplying the *Average* with `1` and adding `0`.
```

Comment:
Not entirely sure what adding zero means, but I imagine this is copied from the substrate repo, so probably not important

Reply:
Yeah this is a copy pasted comment, but yeah I don't really get it as well. Multiply by 1 is just as strange.

Reply:
Yeah this whole comment reads like its a bot or something

Reply:
these are likely "wiggle factors" and offsets they have as inputs for the benchmark output script that generated these stats. They started with higher numbers and eventually decided the benchmark is accurate enough to not need manual "safety buffers"